### PR TITLE
Feature/alcs 1012

### PR DESCRIPTION
**Pull request by lstod**
Legislation to applicant type

### CODE REVIEW

File: bin/migrate-oats-data/applications/app_prep.py [hunk 1]
```diff
@@ -233,6 +242,14 @@ def mapOatsToAlcsAppPrep(data):
 
     return data
 
+def mapOatsToAlcsLeg(data):
```
Comment (mhuseinov) [inline bin/migrate-oats-data/applications/app_prep.py:245]: nitpick: rename to mapOatsToAlcsLegislationCode

File: bin/migrate-oats-data/applications/app_prep.py [hunk 2]
```diff
@@ -233,6 +242,14 @@ def mapOatsToAlcsAppPrep(data):
...
+
+    if data["legislation_code"]:
+        data["legislation_code"] = str(
+            OatsLegislationCodes[data["legislation_code"]].value
+        )
+
+    return data
 
 def get_update_query_for_nfu():
     query = """
```
Comment (mhuseinov) [inline bin/migrate-oats-data/applications/app_prep.py:255]: have you tried to reduce "query" duplication using the constants and custom functions as we discussed or is it in next MR?

Comment (lstod) [reply bin/migrate-oats-data/applications/app_prep.py:255]: Was plan for next MR

**Review by mhuseinov — APPROVED** [review]
🎯

**Pull request merged**